src/main/java/com/endava/soa_5/OOP/Feline.java

```diff
@@ -0,0 +1,57 @@
+package com.endava.soa_5.OOP;
+
+public class Feline extends Mammal implements Ability {
+
+    private final double growingSpeed = 1.36;
+    private String color;
+
+    public Feline(boolean feed, int eye, double weight, double height, String race) {
+        super(feed, eye, weight, height, race);
+    }
+
+    public Feline() {
+        super();
+    }
+
+    public String eat(String product) {
+        if (product == "poison") {
```

Comment:
If you're not using primitive types, use .equals() method.

Reply:
Done!